src/dmd/parse.d

```diff
@@ -8038,6 +8058,22 @@ final class Parser(AST) : Lexer
             e = parseNewExp(null);
             break;
 
+        case TOK.ref_:
+            {
+                if (peekNext() == TOK.leftParentheses)
+                {
+                    Token* tk = peekPastParen(peek(&token));
+                    if (skipAttributes(tk, &tk) && (tk.value == TOK.goesTo || tk.value == TOK.leftCurly))
+                    {
+                        // ref (arguments) => expression
+                        // ref (arguments) { statements... }
+                        goto case_delegate;
+                    }
+                }
+                nextToken();
+                error("found `%s` when expecting function literal following %s", token.toChars(), "`ref`".ptr);
```

Comment:
You can directly put the second variadic argument  in the format specifier.

Reply:
Indeed, I think I read the error a few lines above and copied the format.
What a waste of typing.